src/CommunityToolkit.Maui.UnitTests/Mocks/AnimationExtensions.cs

```diff
@@ -72,7 +72,7 @@ public override async void Start()
 					}
 					catch (Exception e)
 					{
-						Trace.WriteLine(e);
+						Trace.TraceError("{0}", e);
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.